src/components/Display.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = (props) => { // eslint-disable-line
+  return (
+    <>
+      <div>{props.result}</div> {/*eslint-disable-line*/}
+    </>
+  );
+};
```

Comment:
- [ ] `result` should be the incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png)). Then in line 7 just call  {result}